addons/website_event_booth_sale/static/src/js/booth_register.js

```diff
@@ -29,8 +29,8 @@ BoothRegistration.include({
      */
     _updateUiAfterBoothChange(boothCount) {
         this._super.apply(this, arguments);
-        let $elem = this.$('.o_wbooth_booth_total_price');
-        $elem.toggleClass('d-none', !boothCount || !this.categoryPrice);
+        const el = this.el.querySelector(".o_wbooth_booth_total_price");
+        el?.classList.toggle("d-none", !boothCount || !this.categoryPrice);
```

Comment:
@loco-odoo Yes indeed there are cases when `querySelector` is empty, I think since this PR is dependent on [website_event_booth] https://github.com/odoo/odoo/pull/164147) and this method is overriding it, there is an instance when we click 
contact details : 
![image](https://github.com/user-attachments/assets/abce0130-9324-4b9b-8062-294400eddc6e)

the querySelector is empty for this case.